pandas/tests/series/test_timeseries.py

```diff
@@ -129,6 +129,38 @@ def test_shift2(self):
         idx = DatetimeIndex(['2000-01-01', '2000-01-02', '2000-01-04'])
         pytest.raises(NullFrequencyError, idx.shift, 1)
 
+    def test_shift_fill_value(self):
+        # GH #24128
+        ts = Series([1.0, 2.0, 3.0, 4.0, 5.0],
+                    index=date_range('1/1/2000', periods=5, freq='H'))
+
+        # fill_value should have no effect on shift with freq
+        result = ts.shift(1, freq='5T', fill_value=0)
+        exp_index = ts.index.shift(1, freq='5T')
+        tm.assert_index_equal(result.index, exp_index)
```

Comment:
compare actual series

Reply:
@jreback not sure what you mean. the point of this check is to check the index and values independently, can you please specify which series you meant?

Reply:
we are testing series here, why are you even shifting the index? if you are testing indexes they go elsewhere in pandas/tests/indexing, which I don't see at all.

Reply:
ok, I removed the test for index, I just followed the example above the test which did similar checks but without the fill_value